.changelog/2961.feature.md

```diff
@@ -0,0 +1,7 @@
+go/registry/api: Add `GetNodeByConesnusID` method
+
+`GetNodeByConensusID` can be used to query nodes by their Consensus ID.
+
+To query the node by tendermint consensus address use the
+`oasis-core/go/consensus/tendermint/crypto.PublicKeyFromTendermint`
+helper to first convert the tendermint address into a public key.
```

Comment:
This is not actually correct, in case of Tendermint the address is a truncated hash of the public key, so going back is not possible.

Reply:
Ah yeah i missed this, thanks, fixed by instead exposing the `GetNodeByConsensusAddress` method!